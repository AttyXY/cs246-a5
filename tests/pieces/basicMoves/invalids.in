setup
+ K c4
+ k f5 
+ R a1
+ B a8
done
game human human
move a1 a8
move f5 f4
move h8 h7
quit
quit
