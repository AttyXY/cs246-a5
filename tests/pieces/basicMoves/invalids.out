WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  R-------
   abcdefgh

8  B-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  R-------
   abcdefgh

STARTING GAME!

8  B-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  R-------
   abcdefgh

8  B-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  R-------
   abcdefgh

Invalid move. Input another move.
8  B-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  R-------
   abcdefgh

Invalid move. Input another move.
8  B-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  R-------
   abcdefgh

Invalid move. Input another move.
Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
